From cb08843a6f64b862ddf657c2f373884f28e5a26b Mon Sep 17 00:00:00 2001
From: "tao.zeng" <tao.zeng@amlogic.com>
Date: Mon, 23 Jun 2014 19:07:11 +0800
Subject: [PATCH 4380/5965] PD #83628: driver update for AML1218, version 0.90

1. cancel dcdc2 current limit change of register 0x34 according David Wang;
---
 drivers/amlogic/power/aml_pmu/aml1218-sply.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/amlogic/power/aml_pmu/aml1218-sply.c b/drivers/amlogic/power/aml_pmu/aml1218-sply.c
index 7f6ab9036a46..e99b8107e594 100755
--- a/drivers/amlogic/power/aml_pmu/aml1218-sply.c
+++ b/drivers/amlogic/power/aml_pmu/aml1218-sply.c
@@ -1326,7 +1326,7 @@ static int aml1218_update_state(struct aml_charger *charger)
     aml1218_write(0x011f, (chg_status & 0x02000000) ? 0x02 : 0x00);
 
     aml1218_set_bits(0x0035, (chg_status & 0x02000000) ? 0x00 : 0x04, 0x07);
-    aml1218_set_bits(0x003e, (chg_status & 0x02000000) ? 0x00 : 0x04, 0x07);
+    //aml1218_set_bits(0x003e, (chg_status & 0x02000000) ? 0x00 : 0x04, 0x07);
     aml1218_set_bits(0x0047, (chg_status & 0x02000000) ? 0x00 : 0x04, 0x07);
     aml1218_set_bits(0x004f, (chg_status & 0x02000000) >> 22, 0x08);
 
@@ -1499,7 +1499,7 @@ static int aml1218_battery_probe(struct platform_device *pdev)
     int      ret;
     uint32_t tmp2;
 
-	AML1218_DBG("---> PMU driver version:v0.80\n");
+	AML1218_DBG("---> PMU driver version:v0.90\n");
 	AML1218_DBG("call %s in", __func__);
     g_aml1218_init = pdev->dev.platform_data;
     if (g_aml1218_init == NULL) {
-- 
2.19.0

